documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia {
+      # Enable or disable the Jolokia-based metrics/v2 endpoint.
+      # Default is true.
+      enabled: false
+
+      # Configure any of the settings listed at:
+      #   https://jolokia.org/reference/html/agents.html#war-agent-installation
+      config: {
+        # Specify a custom security policy:
+        #  https://jolokia.org/reference/html/security.html
+        policyLocation: "file:///etc/puppetlabs/<service name>/jolokia-access.xml"
```

Comment:
My original take take on the config interface did translate java names like `policyLocation` to Clojure names like `policy-location`. However, I ended up dropping this for the final version because I realized it would pose a documentation problem. We'd either have to re-produce the Jolokia docs with Clojure names and keep it up to date, or explain the translation to folks.

Using `policyLocation` may break with the convention used in the rest of the config file, but it does allow folks to read the docs at https://jolokia.org/reference/html/agents.html#war-agent-installation and apply them directly without any translation or other mental gymnastics.